pom.xml

```diff
@@ -62,7 +63,7 @@
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>bouncycastle-api</artifactId>
-      <version>1.0.3</version>
+      <version>2.16.2</version>
```

Comment:
I hope someone else understands the implications of this.

Reply:
That release version has `<jenkins.version>2.16</jenkins.version>` So I doubt this will work well.

Reply:
Updating to at least bouncycastle-api 1.648 is required for Jenkins > 1.648. Between bouncycastle-api 1.648.3 and 2.16, the Bouncy Castle library (bcpkix) was split from core and moved to the api plugin, so trying to use bouncycastle-api < 2.16.0 with Jenkins >= 2.16.0 means that the Bouncy Castle library is not on the classpath. 

Since we depend on bouncycastle-api in test scope, I think this means that when testing Jenkins < 2.16 both core and the api plugin are both providing a version of bcpkix.

Reply:
I think Devin's interpretation is correct here.